src/microbe_stage/editor/MicrobeEditorCheatMenu.cs

```diff
@@ -5,13 +5,20 @@ public class MicrobeEditorCheatMenu : CheatMenu
     [Export]
     public NodePath? InfiniteMpPath;
 
+    [Export]
+    public NodePath? RevealEntirePatchMapPath;
+
 #pragma warning disable CA2213
     private CustomCheckBox infiniteMp = null!;
+    private Button revealEntirePatchMap = null!;
 #pragma warning restore CA2213
 
     public override void _Ready()
     {
         infiniteMp = GetNode<CustomCheckBox>(InfiniteMpPath);
+        revealEntirePatchMap = GetNode<Button>(RevealEntirePatchMapPath);
+
+        revealEntirePatchMap.Connect("pressed", this, nameof(OnRevealEntirePatchMapClicked));
```

Comment:
Why is this not connected in the scene file?

Reply:
That's how it's done in the other cheat menu, so I figured it would work the same here

Reply:
I must not have been paid close attention then when that was merged. Could you set this in the scene file though as an example of what's the proper way to setup static signal links?